web/src/pages/Proto/components/ProtoDrawer/index.less

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@import '~antd/es/style/themes/default.less';
+
+.formItemEditor {
+  flex-flow: column;
+  row-gap: 20px !important;
+
+  :global {
+    .ant-form-item-explain-error {
+      padding-left: 25px;
+    }
```

Comment:
It seems to have an effect on the error message on all pages here? Please make sure it doesn't affect other form styles.

Reply:
It doesn't affect styles elsewhere, It will only affect the styles inside the "formItemEditor"
<img width="732" alt="image" src="https://user-images.githubusercontent.com/57465570/155256534-118e5aa9-1764-4d4d-84c1-f6b82b37cacf.png">